### PR TITLE
Add helper text to database form fields

### PR DESCRIPTION
**Pull request by ranquild**
Epic: https://github.com/metabase/metabase/issues/19099

- Adds help tooltips for some of the database form fields. 
- Changes copy for the fields.

Screen Shot 2021-11-25 at 10 35 57 pm

How to test:
- Go to Admin > Databases > Create
- Choose different database type and check help tooltips

### CODE REVIEW

File: frontend/src/metabase/entities/databases/big-query-fields.js [hunk 1]
```diff
@@ -69,56 +69,59 @@ export default function getFieldsForBigQuery(details) {
...
+              placeholder: "1w08oDRKPrOqBt06yxY8uiCz2sSvOp3u",
               required: true,
             },
           ]),
       {
         name: "dataset-id",
-        "display-name": "Dataset ID",
-        placeholder: "toucanSightings",
+        "display-name": t`Dataset ID`,
+        "helper-text": t`Make sure to leave out the Project ID prefix in "project_name:dataset_id" and only enter “dataset_id”`,
```
Comment (alxnddr) [inline frontend/src/metabase/entities/databases/big-query-fields.js:81]: Should we change this for the new BigQuery driver as well?

File: frontend/src/metabase/query_builder/components/LimitInput.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
```
Comment (alxnddr) [inline frontend/src/metabase/query_builder/components/LimitInput.jsx:1]: I'm wondering if we can reuse an existing input here with some tweaks if needed

File: frontend/src/metabase/entities/databases/big-query-fields.js (hunk 1 above)
Comment (ranquild) [reply frontend/src/metabase/entities/databases/big-query-fields.js:81]: Thanks! I've updated copy for the new driver. It's on the backend.

File: frontend/src/metabase/query_builder/components/LimitInput.jsx (hunk 2 above)
Comment (ranquild) [reply frontend/src/metabase/query_builder/components/LimitInput.jsx:1]: I'm not sure if we want to. This new `LimitInput` as actually the same as `Input` was before, but it is used only in these 2 places, and it's very different compared to our standard `input` for some reason. I'd leave it as it is for now.

**Review by alxnddr — APPROVED** [review]

**Review by noahmoss — APPROVED** [review]

**Pull request merged**